toolchains/esp32c6/main/toit.c

```diff
@@ -0,0 +1,18 @@
+/* Import Prebuilt Library Example
```

Comment:
Maybe clean these toit.c versions up? The one in toolchains/esp32/main/toit.c looks a lot nicer.